wdio.conf.js

```diff
@@ -47,10 +48,30 @@ const chromeCapabilities = {
   }
 };
 const firefoxCapabilities = {
-  browserName: 'firefox'
+  browserName: 'firefox',
+  'moz:firefoxOptions': {
+    args: [
+      '-start-debugger-server',
+      '9222'
+    ],
+    ...(!isSauceEnabled && {
```

Comment:
We add sauce capabilities at the end, check line 320ish. Would be good to group these changes here.